src/lib/button-toggle/button-toggle.ts

```diff
@@ -377,6 +412,16 @@ export class MatButtonToggle extends _MatButtonToggleMixinBase implements OnInit
   /** Tabindex for the toggle. */
   @Input() tabIndex: number | null;
 
+  /** The appearance style of the button. */
+  @Input()
+  get appearance(): MatButtonToggleAppearance {
+    return this.buttonToggleGroup ? this.buttonToggleGroup.appearance : this._appearance;
+  }
+  set appearance(value: MatButtonToggleAppearance) {
+    this._appearance = value;
+  }
+  private _appearance: MatButtonToggleAppearance;
```

Comment:
Could you add docs for this in a follow-up PR?

Reply:
Sure, I've put it on my list.